docs/reference/builder.md

```diff
@@ -112,7 +112,7 @@ the `Using cache` message in the console output.
 
 Build cache is only used from images that have a local parent chain. This means
 that these images were created by previous builds or the whole chain of images
-was loaded with `docker load`. If you wish to use build cache of a specific
+were loaded with `docker load`. If you wish to use build cache of a specific
```

Comment:
this is not correct either; "the whole chain of images" is also singular (the _chain_ (of images) was loaded with `docker load`)